### ISSUE TITLE
Add Alt Text to Images

### ISSUE THREAD

**Issue by matthewl580** [spindle-project/Website#9]
Description: the project logo image on the page lacks an alt attribute.
To Do:
Add descriptive alt text to all images on the website.
Include relevant keywords in the alt text where appropriate.
Use alt text to provide context for visually impaired users.

**Comment by rico33631** [comment]
Hi @matthewl580 ! I'd love to work on this issue.

Would you like the alt text to follow any specific format or guidelines? Also, are there any images with particular context I should be aware of when writing the descriptions?

**Comment by matthewl580** [comment]
Nice! There aren't any specific guidelines except that the alt text should accurately describe what is on the image.
